huaweicloud/services/dms/resource_huaweicloud_dms_kafka_consumer_group.go

```diff
@@ -0,0 +1,306 @@
+package dms
+
+import (
+	"context"
+	"regexp"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func ResourceDmsKafkaConsumerGroup() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceDmsKafkaConsumerGroupCreate,
+		UpdateContext: resourceDmsKafkaConsumerGroupUpdate,
+		ReadContext:   resourceDmsKafkaConsumerGroupRead,
+		DeleteContext: resourceDmsKafkaConsumerGroupDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"instance_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the ID of the Kafka instance.`,
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the name of the consumer group.`,
+				ValidateFunc: validation.All(
+					validation.StringMatch(regexp.MustCompile(`^[A-Za-z|%-_0-9]*$`),
+						"An instance name starts with a letter and can contain only letters, digits,"+
+							"vertical lines(|), percent sign(%), underscores (_), and hyphens (-)"),
+					validation.StringLenBetween(3, 64),
+				),
```

Comment:
delete the validataion func. it is not used now

Reply:
done